src/resource_clients/actor.js

```diff
@@ -66,36 +66,23 @@ class ActorClient extends ResourceClient {
     async call(options = {}) {
         ow(options, ow.object.exactShape({
             contentType: ow.optional.string,
-            memoryMbytes: ow.optional.number,
-            timeoutSecs: ow.optional.number,
+            memory: ow.optional.number,
+            timeout: ow.optional.number,
             build: ow.optional.string,
             waitSecs: ow.optional.number,
             webhooks: ow.optional.array.ofType(ow.object),
             input: ow.any,
         }));
 
-        const {
-            contentType,
-            memoryMbytes,
-            timeoutSecs,
-            build,
-            waitSecs,
-            input,
-            webhooks,
-        } = options;
-
-        const startOptions = {
-            contentType,
-            waitForFinish: waitSecs,
-            memory: memoryMbytes,
-            build,
-            webhooks,
-            input,
-        };
+        const { timeout, ...callOptions } = options;
+
+        if (timeout >= 0) callOptions.timeout = timeout;
```

Comment:
I'm thinking, instead of this, we could validate that the `timeoutSecs` directly in `ow` to be non-negative.

Reply:
Same for `waitSecs` actually.

Reply:
Added the non-negative validation, removed the condition 👍